libs/ml/include/ml/optimisation/optimiser.hpp

```diff
@@ -16,89 +16,83 @@
 //   limitations under the License.
 //
 //------------------------------------------------------------------------------
-
 #include "math/base_types.hpp"
```

Comment:
optimiser.hpp has been given wrong style indentations - make it impossible to see quickly what changes have been made